config-dev.yaml

```diff
@@ -38,7 +38,9 @@ kafkaStreamsConfig:
 
 topicsConfig:
   producerTopic: _streamregistry_streams_v0.1
-  stateStoreName: _streamregistry_streams_kv_v0.1
+  producerStateStore: _streamregistry_streams_kv_v0.1
+  streamSourceTopic: _stream_source_command_events_v0.1
+  streamSourceStateStore: _stream_source_command_events_store_v0.1
```

Comment:
The config file for the docker image also needs changed

Reply:
Oh. Didn't realize that. Thanks. Will fix.

Reply:
The Docker config uses environment variables, and a lot of them.

The file here is hard coded and is used by the makefile. 

Not sure a good way to use only one

Reply:
Create a file for dev environment variables, source it prior to `make run` and `make debug`, and use the same `config.yaml`?

Reply:
The one for the docker image needs to be located under `src/main/jib/app`